src/modules/Dashboard/features/Managers/Popups/ManagersRemoveAccessPopup/ManagersRemoveAccessPopup.test.tsx

```diff
@@ -16,30 +18,35 @@ const user = {
   email: 'email@gmail.com',
   lastSeen: '',
   roles: [],
-  applets: [],
+  applets: [{ displayName: 'displayName', id: appletId, roles: [] }],
+};
+const commonProps = {
+  onClose: onCloseMock,
+  popupVisible: true,
+  refetchManagers: refetchManagersMock,
+  user,
 };
 
 describe('ManagersRemoveAccessPopup component tests', () => {
+  const mockedAxios = axios.create();
+
   afterEach(() => {
     jest.restoreAllMocks();
   });
 
-  test('ManagersRemoveAccessPopup should appear second screen', async () => {
+  test('ManagersRemoveAccessPopup should open with applets list', async () => {
```

Comment:
please don't use `ManagersRemoveAccessPopup` in each test, it already has declaration in `describe` method